sonar-ws/src/main/java/org/sonarqube/ws/client/WsRequest.java

```diff
@@ -32,8 +32,17 @@
 
   String getMediaType();
 
+  /**
+   *
+   * In case of multi value parameters, returns the first value
+   *
+   * @deprecated since 6.1. Use {@link #getParameters()} instead
+   */
+  @Deprecated
   Map<String, String> getParams();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)